redbpf/src/load/loader.rs

```diff
@@ -108,6 +106,25 @@ pub struct Loaded {
     /// # };
     /// ```
     pub events: mpsc::UnboundedReceiver<(String, <PerfMessageStream as Stream>::Item)>,
+
+    /// Streams of events emitted by BPF ring buffer maps.
+    /// Each key in the map corresponds to a map name.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use std::path::Path;
+    /// use futures::stream::StreamExt;
+    /// use redbpf::load::Loader;
+    /// # async {
+    /// let mut loader = Loader::load_file(&Path::new("probe.elf")).unwrap();
+    /// let mut ringbuf = loader.ringbufs.get("ring_buffer");
+    /// while let Some(event) = ringbuf.next().await {
+    ///     // ...
+    /// }
+    /// # };
+    /// ```
+    pub ringbufs: HashMap<String, RingBufMessageStream>,
```

Comment:
I have a reservation about how this is exposed in the API: there's a discrepancy in how ringbuf and perfmap events are handled.

I think it would make sense to use the same approach for `PerfMap`s too? This would be a breaking change though.

Reply:
As per our agreement with @rhdxmr we are going to bump major version for the next release due to the change in default LLVM. The time to break APIs is now.

Reply:
Sounds good! In this case, I'd advocate for adding a changelog - created an issue #319